### DISCUSSION TITLE
RACINGMODE in EdgeTx

### DISCUSSION THREAD

**Discussion by MiroslavFPV** [EdgeTX/edgetx#1107] [Q&A]
Hello, i played with special functions and found RacingMode there, what is it? What is it doing?

**Comment by raphaelcoeffic** [comment]
@MiroslavFPV this is a FrSky feature, which, as I heard, does not seem to work so well. I was not involved with it’s implementation which is coming from FrSky/OpenTx.

**Comment by pfeerick** [comment]
OTX 2.3.11 => "Frsky: add support for race mode" Hm... looks like it allows telemetry to be disabled "on-demand" / in flight... which I'm sure *everyone* needs. 🤷 What a silly idea, putting a module toggle setting in as a SF...

**Reply by raphaelcoeffic** [reply]
That was my remark at the time, but it was merged against my advise….

**Comment by raphaelcoeffic** [comment]
If nobody opposes this, I think we should remove that "feature". It appears that switching telemetry OFF does not accelerate anything, so that there is, IMHO, no advantage for the user.

**Comment by MiroslavFPV** [comment]
Thanks for reply.

Dňa ne 21. 11. 2021, 9:11 Raphael Coeffic ***@***.***>
napísal(a):
